sshcode.go

```diff
@@ -548,6 +564,11 @@ chmod +x %v`,
 func ensureDir(path string) error {
 	_, err := os.Stat(path)
 	if os.IsNotExist(err) {
+		// This fixes a issue where Go reads `/c/` as `C:\c\` and creates
+		// empty directories on the client that don't need to
```

Comment:
```suggestion
		// empty directories on the client that don't need to exist.
```

Reply:
oh! gods above that was a stupid mistake. will fix next push

Reply:
This needs a period as well at the end.

Reply:
Well i cant add that for some stupid reason. that single period COMPLETELY FUCKS my local repo (even after deleting and re-cloning to make the edit
```
[user@LATITUDE Workspace]$ hub clone merith-tk/sshcode
Cloning into 'sshcode'...
remote: Enumerating objects: 50, done.
remote: Counting objects: 100% (50/50), done.
remote: Compressing objects: 100% (40/40), done.
remote: Total 440 (delta 29), reused 27 (delta 10), pack-reused 390
Receiving objects: 100% (440/440), 815.42 KiB | 1.17 MiB/s, done.
Resolving deltas: 100% (247/247), done.
[user@LATITUDE Workspace]$ cd sshcode/
[user@LATITUDE sshcode]$ hub checkout msys-support
Branch 'msys-support' set up to track remote branch 'msys-support' from 'origin'.
Switched to a new branch 'msys-support'
[user@LATITUDE sshcode]$ hub add .
[user@LATITUDE sshcode]$ hub commit --amend -m "fix typo"
[msys-support 8941d12] fix typo
 Date: Sat Apr 18 12:28:37 2020 -0700
 1 file changed, 1 insertion(+), 2 deletions(-)
[user@LATITUDE sshcode]$ hub push
To github.com:Merith-TK/sshcode.git
 ! [rejected]        msys-support -> msys-support (non-fast-forward)
error: failed to push some refs to 'git@github.com:Merith-TK/sshcode.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

Reply:
found out what i was doing wrong. i was trying to amend the commit ... yeah not good